autopush/webpush_server.py

```diff
@@ -434,58 +431,6 @@ def _validate_chid(chid):
     return True, None
 
 
-@attrs(slots=True)
-class Register(InputCommand):
-    channel_id = attrib()  # type: str
-    uaid = attrib(convert=uaid_from_str)  # type: Optional[UUID]
-    message_month = attrib()  # type: str
-    key = attrib(default=None)  # type: str
-
-
-@attrs(slots=True)
-class RegisterResponse(OutputCommand):
-    endpoint = attrib()  # type: str
-
-
-@attrs(slots=True)
-class RegisterErrorResponse(OutputCommand):
-    error_msg = attrib()  # type: str
-    error = attrib(default=True)  # type: bool
-    status = attrib(default=401)  # type: int
-
-
-class RegisterCommand(ProcessorCommand):
-
-    def process(self, command):
-        # type: (Register) -> Union[RegisterResponse, RegisterErrorResponse]
-        valid, msg = _validate_chid(command.channel_id)
```

Comment:
We have _validate_chid here to mimick websocket.py's validation of the channel_id sent in the register command is hyphenated.

Now I'm wondering if this check is bogus: because the rust side parses the register command now. It's deserializing it via serde into ClientMessage::Register channel_id: Uuid. Does serde's deserialization allow non hyphenated uuids to pass through unnoticed?

Reply:
Yes, serde will allow hypenated or not to pass through.